src/services/balancer/subgraph/entities/poolActivities/index.ts

```diff
@@ -1,12 +1,15 @@
-import Service from '../../balancer-subgraph.service';
+import { BalancerSubgraphService } from '../../balancer-subgraph.service';
```

Comment:
What's the reasoning behind removing the default export here?

Reply:
Mostly a personal preference. When working with `TypeDI` I prefer to know exactly which service one is referring to. It is also my opinion that `Service` - as exported here - is not clear enough if we introduce the formal concept of services through `TypeDI`.